model/profile/common.yaml

```diff
@@ -6,3 +6,5 @@ groups:
     attributes:
       - ref: profile.frame.type
         requirement_level: recommended
+      - ref: profile.pprof.mapping
```

Comment:
I noticed that this `profile.pprof.mapping` attribute is under `profile.frame group` which seems a little odd.

Reply:
Created dedicated group with `profile.pprof` and moved `profile.pprof.mapping` to it - see https://github.com/open-telemetry/semantic-conventions/pull/2522/commits/d1708e792880b7b769bb6752317fd863dd2f9321